tests/utils.py

```diff
@@ -213,12 +219,12 @@ def _start_process(self, extra_cmd_args, popen_cwd):
 
 
 
-  def _start_redirect_thread(self):
+  def _start_redirect_thread(self) -> None:
     """Starts a thread responsible to redirect stdout/stderr to the Queue."""
 
     # Run log_queue_worker() in a thread.
     # The thread will exit when the child process dies.
-    self._log_queue = queue.Queue()
+    self._log_queue: queue.Queue = queue.Queue()
```

Comment:
unlikely to need annotation?

Reply:
I received a warning before about that, but I can't reproduce it now.
So, yea removing it.